app/controllers/support_ticket/check_your_request_controller.rb

```diff
@@ -8,11 +8,12 @@ def new
   end
 
   def save
-    if form.valid?
-      form.create_ticket
+    if form.valid? && form.create_ticket
       session[:support_ticket] = nil
+      session[:support_ticket_number] = form.ticket_number
```

Comment:
non blocker but does this need to be deleted later? or can it hang about? which brings me to a possible solution that using `flash` might be a better option

Reply:
I don't think this needs to be deleted because if they navigate to another page and decide to use the browser back button then the page will still load up as expected.

If they go through the get support journey again then the session value would be updated